include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -372,6 +375,12 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Normalize,Clamp,Sw
 				return core::vectorSIMDi32(kernelX.getWindowMinCoord(halfTexelOffset).x-1,kernelY.getWindowMinCoord(halfTexelOffset).y-1,kernelZ.getWindowMinCoord(halfTexelOffset).z-1,0);
 			}();
 			const auto windowMinCoordBase = inOffsetBaseLayer+startCoord;
+
+			const core::vectorSIMDu32 phaseCount = getPhaseCount(inExtentLayerCount, outExtentLayerCount);
+
+			const size_t maxPhaseSupportLUTSize = core::max(phaseCount[0], core::max(phaseCount[1], phaseCount[2])) * core::max(kernelX.getWindowSize()[0], core::max(kernelY.getWindowSize()[1], kernelZ.getWindowSize()[2])) * sizeof(double);
+			state->phaseSupportLUT = core::make_smart_refctd_ptr<ICPUBuffer>(maxPhaseSupportLUTSize);
```

Comment:
compute and prepare in a `CState` method (so you can reuse for multiple `execute` calls)

this also requires the LUT to be `sum(phaseCount[i]*kernel[i].getWindowSize()[i])` instead of `max_element(phaseCount[i]*kernel[i].getWindowSize()[i])`

Reply:
Resolved.